content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md

```diff
@@ -25,6 +25,13 @@ For example:
 confidentialIdentityMinimumBackupInterval = 10m
 ```
 
+or
+
+```
+confidentialIdentityMinimumBackupInterval = 2d

```

Comment:
I would switch these values to like 5d and 30d (i think 10m is like a value for this param that no one would ever use)